src/Page/AboutPage/index.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "./styles.css";
+import backgroundImg from "./assets/section-bg.png";
+
+function AboutPage() {
+  return (
+    <main className="about-page">
+      <div className="page-content">
+        <section className="page-section d-flex flex-column mb-0">
+          <h3 className="section-head mb-3">Our Story</h3>
+          <hr className="heading-hr mb-3 align-self-center" />
+          <div>
+            <img src="https://nightingale1.s3.ap-south-1.amazonaws.com/images/section-bg.png" className="backgroundImg" />
+          </div>
+          <div className="about-page-content d-flex flex-column justify-content-center align-items-center">
+            <p className="w-50 section-para">
+              Started in 2016, Navgurukul runs one-year residential course for
```

Comment:
Maybe consider introducing constants for paragraphs of text so you could easily toggle between languages later.